python/doc/theory/numerical_methods/nataf_transformation.rst

```diff
@@ -3,40 +3,35 @@
 Generalized Nataf Transformation
 --------------------------------
 
-The Generalized Nataf transformation is an isoprobabilistic
-transformation (refer to ) which is used under the following context :
-:math:`\vect{X}` is the input random vector, :math:`F_i` the
-cumulative density functions of its components and :math:`C` its
-copula, which is supposed to be elliptical.
+The Generalized Nataf transformation is an :ref:`isoprobabilistic transformation <isoprobabilistic_transformation>`  which is
```

Comment:
This comment is on the wrong place, but I cannot reach the "References" section below, because you did not modify it.

The references are poorly formatted:

![image](https://user-images.githubusercontent.com/31351465/235197069-02c0b2df-c0d9-44d7-9c0e-be4b9f35a866.png)

This is because the references are written on a single line that RST cannot format.

If we split the line and write the remaining lines of each item aligned with the first:

```
.. topic:: References:

    - O. Ditlevsen and H.O. Madsen, 2004, "Structural reliability methods," 
      Department of mechanical engineering technical university of Denmark - 
      Maritime engineering, internet publication.
    - J. Goyet, 1998, "Sécurité probabiliste des structures - 
      Fiabilité d'un élément de structure," Collège de Polytechnique.
    - A. Der Kiureghian, P.L. Liu, 1986,"Structural Reliability Under 
      Incomplete Probabilistic Information", Journal of Engineering Mechanics, 
      vol 112, no. 1, pp85-104.
    - [lebrun2009a]_
    - [lebrun2009b]_
    - [lebrun2009c]_
    - H.O. Madsen, Krenk, S., Lind, N. C., 1986, "Methods of Structural 
      Safety," Prentice Hall.
    - [nataf1962]_

```
it produces:
![image](https://user-images.githubusercontent.com/31351465/235195475-0dabd7f0-d417-4c56-9e1e-f80512b46bd4.png)

Reply:
ok....